problems/1.types+operators.js

```diff
@@ -2,15 +2,25 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
+function isNumber(val) {
+  if (typeof(val) === 'number'){
```

Comment:
Can get away with just returning the condition. Don't need parenthesis for argument of typeof